client/components/ProductHistory.jsx

```diff
@@ -0,0 +1,53 @@
+import React, {Component} from 'react'
+import axios from 'axios'
+import {connect} from 'react-redux'
+
+class ProductHistory extends Component {
+	constructor(){
+		super()
+		this.state = {
+			orders : []
+		}
+	}
+
+	componentDidMount() {
+		axios.get(`/api/users/${this.props.user.id}/orders`)
```

Comment:
This route would be a problem later, when we implemented unauthenticated users.

Reply:
feels like axios call shouldn't be in the component, because we want to keep component simple. It should be a thunk.

Reply:
it shouldnt be a thunk because we're not indicating a intent to change the store. because our orders isnt part of the state.

We can make it a dumb component later by passing props to this component. 

We can just be defensive later. Un-auth users wont have a history, so we can just not make the axios call at all.

Reply:
I think if we know it's going to be a problem later, we should leave the loop hole in there, cuz later, we might not remember, and have to debug again.
I think it's an easy fix. Just check if global state user.id, if there's no id, don't make that axios call.